File: reference.vb.net/Db4objects.Db4odoc/Refactoring/PilotNew.vb
' Copyright (C) 2004 - 2006 db4objects Inc. http://www.db4o.com 
Imports System

Namespace Db4objects.Db4odoc.Refactoring
    Public Class PilotNew
        Private _identity As String
        Private _points As Integer

        Public Sub New(ByVal name As String, ByVal points As Integer)
            _identity = name
            _points = points
        End Sub

        Public ReadOnly Property Identity() As String
            Get
                Return _identity
            End Get
        End Property

        Public Overrides Function ToString() As String
            Return String.Format("{0}/{1}", _identity, _points)
        End Function
    End Class
End Namespace

File: reference.net/db4o-reference-chapters/src/Db4objects.Db4odoc/Refactoring/PilotNew.cs
/* Copyright (C) 2004 - 2007 db4objects Inc. http://www.db4o.com */
namespace Db4objects.Db4odoc.Refactoring
{
	public class PilotNew
	{
		private string _identity;
		private int _points;
		
		public PilotNew(string name, int points) 
		{
			_identity = name;
			_points = points;
		}

		public string Identity
		{
			get 
			{
				return _identity;
			}
		}

		override public string ToString() 
		{
			return string.Format("{0}/{1}",_identity,_points);
		}
	}
}
